ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### Prerequisites
+
+We thank you for reporting a bug or feature request and helping us to make the SeqAn library better.
+We would like you to check the following points before continuing with the submission as often
+some items might have been solved already. 
+
+* [ ] Are you running the latest version?
+* [ ] Did you check the debugging guide?
+* [ ] Did you check the FAQs on Discuss?
+* [ ] Did you perform a cursory search?
```

Comment:
What do you mean? Search the older issues? As a user I wouldn't know what you want from me..

Reply:
well yes, cursory search means something like did you do a quick search on the repository to find something related. Like a keyword search regarding the problem.

Reply:
Ok. I would add this explanation because I think it is not clear (Maybe its just me but I think cursory is not a word everybody knows). Maybe rather: `Did you look for related issues in our repository?`